bolt/lib/Rewrite/RewriteInstance.cpp

```diff
@@ -237,6 +237,12 @@ UseGnuStack("use-gnu-stack",
   cl::ZeroOrMore,
   cl::cat(BoltCategory));
 
+static cl::opt<uint64_t> CustomAllocationVMA(
```

Comment:
Do you want this flag to be at the top level or hidden?

Reply:
Yeah, it's a good idea to put it hidden. Let me do that.